Site/webapp/js/client/controllers/AccessRequestView.jsx

```diff
@@ -1,11 +1,59 @@
-import React, { Component, Fragment } from 'react';
+import React, { Component, Fragment, useEffect, useState } from 'react';
 
 import { Link } from '@veupathdb/wdk-client/lib/Components'
 
 import SupportFormBase from '@veupathdb/web-common/lib/components/SupportForm/SupportFormBase';
 import SupportFormBody from '@veupathdb/web-common/lib/components/SupportForm/SupportFormBody';
 
-export default class AccessRequestView extends Component {
+import { useWdkService } from '@veupathdb/wdk-client/lib/Hooks/WdkServiceHook';
+import { useStudyAccessApi } from '@veupathdb/study-data-access/lib/study-access/studyAccessHooks';
+
+const camelToSnakeCase = (str) =>
+  str.replace(/[A-Z]/g, (letter) => `_${letter.toLowerCase()}`);
+
+const AccessRequestView = (props) => {
+  const [existingRequestData, setExistingRequestData] = useState(null);
+  const user = useWdkService((wdkService) => wdkService.getCurrentUser(), []);
+  const studyAccessApi = useStudyAccessApi();
+
+  useEffect(() => {
```

Comment:
Could you add a comment explaining what data is being fetched?

Reply:
Yep, good idea!